### DISCUSSION TITLE
404怎么办啊

### DISCUSSION THREAD

**Discussion by huan201** [Lost-MSth/Arcaea-server#163] [Q&A]
![IMG_20240504_133753](https://github.com/Lost-MSth/Arcaea-server/assets/88644963/eaf0730d-5169-4237-8116-6da3b52339e1)
![Uploading IMG_20240504_133744.jpg…]()
![Uploading IMG_20240504_133733.jpg…]()
这个该怎么办啊

**Comment by Lost-MSth** [comment]
后面图没传上来，但根据 404 和第一张图可以判断，应该是没设置 `GAME_API_PREFIX` 吧，请仔细看看文档。
